### PR TITLE
Consolidate main.css and main-mod.css

### LINKED ISSUE #5: Consolidate main.css and main-mod.css
Why do we have 2 different main csses? Most of it is similar. Squish them. Some pages only have main, and some others only have main-mod. This can lead to confusion and inconsistent style.

### PR DESCRIPTION
**Pull request by rolsanmordua**
Closes #5

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -1778,7 +1778,8 @@
 	}
 
 	h1, h2, h3, h4, h5, h6 {
-		color: #444;
+		color: #c15b05;
+		/* color: #444 */
```
Comment (ChiaHuaLu) [inline assets/css/main.css]: Should we delete this altogether?

File: assets/css/main.css [hunk 2]
```diff
@@ -1851,7 +1852,8 @@
 		input[type="button"].icon,
 		button.icon,
 		.button.icon {
-			padding-left: 1vw;
+			/* padding-left: 1vw; */
```
Comment (ChiaHuaLu) [inline assets/css/main.css]: delete this altogether?

File: assets/css/main.css [hunk 3]
```diff
@@ -40,7 +40,7 @@
 	}
 
 	table {
-		border-collapse: collapse;
+		/* border-collapse: collapse; */
```
Comment (ChiaHuaLu) [inline assets/css/main.css]: remove this altogether?

File: assets/css/main.css [hunk 4]
```diff
@@ -4399,7 +4412,7 @@
 	}
 
 	#footer-iframe {
-	height: 7em;
+	/* height: 8em; */
```
Comment (ChiaHuaLu) [inline assets/css/main.css]: do we need this class still?

**Comment by rolsanmordua** [comment]
Made changed based on code review and found new issue with mobileheader.html (fixed).

**Pull request merged**